src/screens/EncounterHomeScreen.tsx

```diff
@@ -67,7 +66,7 @@ export const EncounterHomeScreen = ({ navigation, route }: any) => {
   const error = errorQr || errorUser;
   const refreshing = refreshingQr || refreshingUser;
 
-  const qrValue = Linking.createURL('encounter', { queryParams: { qrId } });
+  const qrValue = Linking.createURL(HOST_NAMES.ENCOUNTER, { queryParams: { qrId } });
 
   const onPressInfo = useCallback(() => {
```

Comment:
I am getting this error for `useSelectImage` in `EncounterRegistrationScreen`.

![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 09 34 27](https://github.com/user-attachments/assets/f37a5f40-df5c-41ed-9e2d-2f5c8cfb0bc6)

Reply:
Fixed, please try again with latest code

![Simulator Screenshot - iPhone 15 Plus - 2024-09-19 at 15 39 07](https://github.com/user-attachments/assets/01882a14-bcf5-4a73-b4ce-f48c53e9497c)